source/tools/testConsistency/dotnetWPF-renderer/test_app.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Microsoft Corporation. All rights reserved. 
+# Licensed under the MIT License. 
+import unittest
+import json
+from app import app
+from copy import deepcopy
+
+
+POST_URL = 'http://127.0.0.1:5000/'
+TEST_PARTIAL_RESULT_URL = 'http://127.0.0.1:5000/test'
+
+class TestFlaskApi(unittest.TestCase):
```

Comment:
>TestFlaskApi [](start = 6, length = 12)

Do add comments in the code explaining what the general purpose of this class is and also add names/descriptions to the actual tests themselves to make it very self-evident say in a run log as to what the test is and thus what it is that it's testing.

Reply:
So each test has a description. There is a general way of running python unit tests which is 
`python -m unittest -v` in the directory that contains the testing script. This will run all the tests along with descriptions as to what is being tested